packages/babel-register/src/node.js

```diff
@@ -42,7 +42,7 @@ function compile(code, filename) {
     // sourceRoot can be overwritten
     {
       sourceRoot: path.dirname(filename) + path.sep,
-      ...deepClone(transformOpts),
+      ...cloneDeep(transformOpts),
```

Comment:
And the reason we probably want to keep this is because `transformOpts` might have like things like functions vs. just objects given how people configure @babel/register with js?

Reply:
Yeah, also people can pass any JS value as plugin options.